docs/cli-reference.md

```diff
@@ -474,10 +500,16 @@ The URLs that are generated using this command, can be used to access the deploy
   # Zsh terminal PS1 support
   source <(odo utils terminal zsh)
 
+
+   # Set a configuration value
```

Comment:
@kadel This is auto-generated though so it may have been an issue of no one updating the cli-reference from before @syamgk  made changes / auto-generated it.